docs/configuration.md

```diff
@@ -45,6 +45,7 @@ Unless you need to set a non-default value, it is recommended to only populate o
 
 | Name                                  | Default       | Description                                                                                                                                                                                                                                                                                            |
 | ------------------------------------- | ------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `ALLOW_INVALID_HEX_RESPONSE`          | "true"        | Account for the benign 400 response with message 'data field invalid hexadecimal string' coming back from the mirror node when deploying a contract.
```

Comment:
Need to update this
`ESTIMATE_GAS_THROWS` => Something to describe that this decided is `eth_estimateGas` throws a revert error on execution reverts.

Reply:
Feel free to rename if `ESTIMATE_GAS_THROWS` isn't descriptive enough and needs to be `ESTIMATE_GAS_EXPOSES_REVERT` or something else instead.

Reply:
Updated.